About Loca

```diff
@@ -0,0 +1,18 @@
+
+
+                                                About Loca
+
+Why pay hundreds of dollars for an overpriced, problematic and confusing rental property management software, when you can create a similar app to your taste and preferences, without any  of the complex or complicated features… and best of all, absolutely free to use anytime, anywhere?
+
+This is the underlying philosophy behind the creation and development of Loca by its Lead Developer, Camel Aissani. A rental property manager by profession who also enjoys Software Development and Coding as a part time hobby.
+
+From the initial plan of creating Loca to help himself and his fellow rental property manager cum friend in their businesses, Loca has evolved into  a fully integrated web-based, interactive and no-cost solution.
+
+Loca is currently helping thousands of rental estate professionals globally to effectively manage their businesses, keep accurate and secure rental records, significantly reduce the need for making frequent, costly phone calls to tenants or vendors and avoid most of the tiresome paperwork associated with managing a rental property.
```

Comment:
Unfortunately, I cannot confirm this point. I don't have any metrics to count the number of people using Loca.